coursetracker/views.py

```diff
@@ -29,10 +27,4 @@ def course(request, course_name):
         print(f"*Course {course_name} does not exist")
         return render(request, 'coursetracker/404.html')
 
-    sections_teaching_team = json.loads(c.sections_teaching_team)
-    professor_ratings = json.loads(c.professor_ratings)
-    preq_tree = json.loads(c.prerequisite_tree)
-
-    return render(request, 'coursetracker/course.html', {'course': c, 'preq': preq_tree,
-                                                         'sections_teaching_team': sections_teaching_team,
-                                                         'professor_ratings': professor_ratings})
+    return render(request, 'coursetracker/course.html', {'course': c, 'preq': c.prerequisite_tree})
```

Comment:
@alvinkam33  I am trying to make the dictionary in this line only `{'course': c}`, but I can't get the prerequisite tree to work when replacing `preq` with `course.prerequisite_tree` in tree.html. I think it might be because it recursively calls tree.html? Can you see if it is possible to solve this issue? Thanks

Reply:
@patrick-5546 Working now. `views.py` only passes in {'course': c} into `course.html`